.github/workflows/master.yml

```diff
@@ -72,3 +72,26 @@ jobs:
       run: ./dev/check-license
     - name: Dependencies
       run: ./dev/test-dependencies.sh
+
+  lintr:
+    runs-on: ubuntu-latest
+    name: Linter (R)
+    steps:
+    - uses: actions/checkout@master
+    - uses: actions/setup-java@v1
+      with:
+        java-version: '11'
+    - name: install R
+      run: |
+        echo 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' | sudo tee -a /etc/apt/sources.list
+        sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
+        sudo apt-get update
+        sudo apt-get install -y r-base r-base-dev libcurl4-openssl-dev
+    - name: install R packages
+      run: |
+        sudo Rscript -e "install.packages(c('curl', 'xml2', 'httr', 'devtools', 'testthat', 'knitr', 'rmarkdown', 'roxygen2', 'e1071', 'survival'), repos='https://cloud.r-project.org/')"
+        sudo Rscript -e "devtools::install_github('jimhester/lintr')"
```

Comment:
@dongjoon-hyun, how about we set the specific commit so we prevent unexpected lint failure by the latest changes from lintr?

Also, out of curiosity, is there reason we install it here instead of:

https://github.com/apache/spark/blob/02c91e03f975c2a6a05a9d5327057bb6b3c4a66f/dev/lint-r.R#L30

?

Reply:
Oh, they finally released 2.0.0 (https://github.com/jimhester/lintr/releases/tag/v2.0.0). We should better match the version to it now.

Reply:
Yes.  :)

Reply:
The `lint-r.R` will be ignored in GitHub Action environment.

Reply:
I didn't remove that because we still have Jenkins and other environments which we cannot install R and packages. We only can handle GitHubAction and AppVeyor.

Reply:
Ah then it's fine - might be good to leave a short comment for this duplication. Can we just use a version tag 2.0.0 instead for here and `spark/dev/lint-r.R`?

Otherwise Looks fine.

Reply:
Of course!

Reply:
Jenkins one is already pretty old. Currently lintr isn't self-contained sadly .. (see https://github.com/apache/spark/pull/19959)

I actually faced the same concern at https://github.com/apache/spark/pull/19290

Since it installs that version only when lintr is unavailable:

https://github.com/apache/spark/blob/02c91e03f975c2a6a05a9d5327057bb6b3c4a66f/dev/lint-r.R#L29

it might be fine

Reply:
Thank you for checking. Yes. It does.